First-outlook.md

```diff
@@ -8,33 +8,32 @@ Things you should try out.
 
 ### ⚠️ Common pitfalls + FAQ
 
--> [Unofficial Heaps FAQ](https://gist.github.com/Yanrishatum/ae3725a9e2b45e0766c065e573ed1f24#an-unofficial-heaps-faq)
+[Unofficial Heaps FAQ](https://gist.github.com/Yanrishatum/ae3725a9e2b45e0766c065e573ed1f24#an-unofficial-heaps-faq)
 
 ### ⭐️ Github samples
 
-Besides the [samples you can run online on the Heaps site](https://heaps.io/samples/) you should try the many [**samples from the github repository**](https://github.com/HeapsIO/heaps/#samples). There are many samples that haven't yet been added to the site of Heaps. The samples are already in your Haxe's Heaps folder on your local machine: `HaxeToolkit\haxe\lib\heaps\git\samples`. When compiling the `all.hxml` file all samples should get compiled (which currently takes about 227 MB of disk space).
+Besides the [samples you can run online on the Heaps site](https://heaps.io/samples/) you should try the many [**samples from the github repository**](https://github.com/HeapsIO/heaps/#samples).
 
-Sidenote (Windows): When using HashLink you can run them very quickly when [adding HashLink to your right mouse menues](https://github.com/HaxeFoundation/hashlink/wiki/Further-Tips#working-on-windows).
+There are many samples that haven't yet been added to the site of Heaps. The samples are already in your Haxe's Heaps folder on your local machine: `HaxeToolkit\haxe\lib\heaps\git\samples`. 
```

Comment:
I think this is out of date now since the website is now deploying again

Reply:
> I think this is out of date now since the website is now deploying again

Not sure tbh. Some of the samples right now are erroring in the heaps repo and are not being shown on the website. I'll revisit this section once I fix the samples that are erroring.

Reply:
I think only samples listed in https://github.com/HeapsIO/heaps.io/blob/master/assets/content/samples/samples.json are on the website, which is probably not all of them.